generators/app/templates/src/StarterKit/StarterKit.csproj

```diff
@@ -26,16 +25,17 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="AutoMapper" Version="8.1.1" />
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="6.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.5" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Versioning" Version="4.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Versioning.ApiExplorer" Version="4.1.1" />
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="2.2.4" />
-    <PackageReference Include="Digipolis.Web" Version="9.1.0" />
     <PackageReference Include="Digipolis.DataAccess" Version="4.0.0" />
     <PackageReference Include="Digipolis.ApplicationServices" Version="2.0.0" />
     <PackageReference Include="Digipolis.Common" Version="3.0.0" />
     <PackageReference Include="Digipolis.Correlation" Version="5.3.1" />
-    <PackageReference Include="Digipolis.Errors" Version="5.2.0" />
+    <PackageReference Include="Digipolis.Errors" Version="7.0.0" />
     <PackageReference Include="Digipolis.Json" Version="3.0.0" />
```

Comment:
Volgens mij mag deze er al uit, het is de bedoeling deze dependency er uit te werken